HOW_WE_USE_GITHUB.md

```diff
@@ -238,17 +238,189 @@ please post details to the [Nucleus forums](https://community.anaconda.cloud/).
+
+#### Task spikes
+
+Task spikes belong to an existing issue. Common use cases for these types of spikes is
+determining if a feature request is viable and if we want to pursue it further, or if a
+reported bug is worth addressing or how best to address it.
+
+When creating a task spike, do the following:
+
+- Under the issues' "tasks" (sometimes called "What should be done?"), add a description
+  of the spike and what needs to be done
+- Add a timebox next to this task (either "low", "medium" or "high")
+- If necessary, add further sub-tasks one level underneath the spike task
+
+Here's an example of what that looks like:
+
+---
+
+- [ ] Create a proposed fix for this bug  (timebox: low)
+  - [ ] Create a pull request with your proposed approach
+  - [ ] Justify and explain your approach on this pull request
+
+---
+
+For the example above, the contributor will create a prototype for a bug fix that will later
+be discussed with the project team or the wider conda community. The authors of the spike have
+specified "low" as the timebox level, which indicates that they do not want to spend too much
+time researching this issue.
+
+If work is still ongoing after reaching the timebox limit specified, the contributor is encouraged
+to contact the spike authors to report their findings to see if further work is desired.
+
+#### Issue spikes
+
+Issue spikes are much like task spikes except that they have enough work to warrant an entire
+issue to themselves. These are often created when planning new features or investigating new
+technologies that could benefit the project team.
+
+When creating an issue spike, do the following:
+
+- Add the "spike" label to the issue
+- Add a "Tasks List" section with a list of all applicable tasks for the spike.
+  For example, if we expect to create another issue as a result of this spike, that should be stated
+  there.
+
+Here is an example issue spike:
+
+---
+
+**Description**
+
+In order to better understand the needs of conda users, we intend to create a prototype for a new
+feature.
+
+**Timebox:** High
+
+**Task List**
+
+- [ ] Create a technical specification for this new feature
+- [ ] Create a pull request demonstrating how this feature works with an example use case of it
+- [ ] Present these findings at the upcoming conda community meeting
+
+---
+
+The example above outlines all the steps necessary in order to create a prototype to present to
+the conda community or project team. For this particular spike, the authors have specified its
+timebox as "high", so as a contributor, you should plan to spend a considerable amount of time
+on this spike.
+
+As someone working on a spike such as this, you are also highly encourage to keep all applicable
+information and discussions on this issue. This helps encourage a culture of asnychronous communication,
+and also helps us keep public records of important conversations that can be easily referenced later.
+
+#### Additional guidelines to be aware of
```

Comment:
This header should either be:
`#### Additional guidelines to be aware of:`
or 
`#### Additional guidelines`